CHANGES.md

```diff
@@ -23,6 +23,38 @@ OpenSSL 3.0
 
 ### Changes between 1.1.1 and 3.0 [xx XXX xxxx]
 
+ * OSSL_STORE_INFO_get_type() may now return an additional value. In 1.1.1
+   this function would return one of the values OSSL_STORE_INFO_NAME,
+   OSSL_STORE_INFO_PKEY, OSSL_STORE_INFO_CERT or OSSL_STORE_INFO_CRL. In 3.0
```

Comment:
It could have and still can return also OSSL_STORE_INFO_PARAMS.

Reply:
It should be pointed out that in 1.1.1, a decoded *public key* would still have the info type `OSSL_STORE_INFO_PKEY`.  `OSSL_STORE_INFO_PUBKEY` was introduced to inform the application what sort of key it really got.